From e48f73d6a5e90d2f883e15ccedf4f53d26bb6e74 Mon Sep 17 00:00:00 2001
From: Olaf Rempel <razzor@kopf-tisch.de>
Date: Wed, 9 Nov 2005 15:25:40 +0100
Subject: [PATCH] iproute2-2.6.14-051107: missing arpd directory

arpd requires a directory (/var/lib/arpd/) to run.
see attached patch, which lets iproute create this directroy during install.
---
 Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff -up iproute-2.6.29/iproute2-2.6.29/Makefile.arpd iproute-2.6.29/iproute2-2.6.29/Makefile
--- iproute2-2.6.29/Makefile.arpd	2009-04-14 12:57:23.743114366 +0200
+++ iproute2-2.6.29/Makefile	2009-04-14 12:58:38.754111501 +0200
@@ -5,6 +5,7 @@ CONFDIR=/etc/iproute2
 DOCDIR=/share/doc/iproute2
 MANDIR=/share/man
 KERNEL_INCLUDE=/usr/include
+ARPDDIR=/var/lib/arpd                                                                       
 
 # Path to db_185.h include
 DBM_INCLUDE:=/usr/include
@@ -44,6 +45,7 @@ Config:
 install: all
 	install -m 0755 -d $(DESTDIR)$(SBINDIR)
 	install -m 0755 -d $(DESTDIR)$(CONFDIR)
+	install -m 0755 -d $(DESTDIR)$(ARPDDIR)
 	install -m 0755 -d $(DESTDIR)$(DOCDIR)/examples
 	install -m 0755 -d $(DESTDIR)$(DOCDIR)/examples/diffserv
 	install -m 0644 README.iproute2+tc $(shell find examples -maxdepth 1 -type f) \
